pkg/storage/redis/client_test.go

```diff
@@ -9,6 +9,43 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestParseClusterURL(t *testing.T) {
+	opts, err := ParseClusterURL("redis+cluster://CLUSTER_USERNAME:CLUSTER_PASSWORD@localhost:26379,otherhost:26479/?" + (&url.Values{
+		"read_only":            {"true"},
+		"username":             {"USERNAME"},
+		"password":             {"PASSWORD"},
```

Comment:
maybe we should complain if people provide unsupported parameters, instead of silently ignoring them?